goth/runner/__init__.py

```diff
@@ -247,15 +232,12 @@ async def __call__(
             self._test_failure_callback(err)
 
     async def _enter(self) -> None:
-        logger.info("Running test: %s", self._get_current_test_name())
```

Comment:
I don't think we should remove this log message as it proves to be helpful when debugging test failures. I'd suggest moving the test name extraction logic to a utility function and use it here (see also a related comment in `test/yagna/conftest.py`).